test/dsl/dsl_basic_model_construction.jl

```diff
@@ -71,7 +71,7 @@ let
           Set([:p, :k1, :k2, :k3, :k4, :k5, :k6, :d])
     basic_test(reaction_networks_hill[1], 4, [:X1, :X2],
                [:v1, :v2, :K1, :K2, :n1, :n2, :d1, :d2])
-    basic_test(reaction_networks_constraint[1], 6, [:X1, :X2, :X3],
+    basic_test(reaction_networks_conserved[1], 6, [:X1, :X2, :X3],
```

Comment:
I named the `reaction_networks_constraint` networks that when I didn't really know about the terminology that well. Changde the names to something better.